src/Promitor.Agents.Scraper/Validation/RuntimeValidator.cs

```diff
@@ -39,26 +38,27 @@ public class RuntimeValidator
             };
         }
 
-        public void Run()
+        public bool Run()
```

Comment:
Maybe return an enum instead or the list itself?

Returning a bool for a `Run` method feels a bit odd, but not sure how to approach it otherwise than the above 🤔

Reply:
Yeah - I was half considering doing that, or renaming the method, but in the end I opted to just minimise the changes, but since we both had the same thought I'll rework it.